### PR TITLE
fix some typo in script files

### PR DESCRIPTION
**Pull request by oliviershrek**
Fixes #

## Proposed Changes

*
*
*

**Release Note**

```release-note

```

### CODE REVIEW

File: hack/init-primary-config.sh [hunk 1]
```diff
@@ -23,8 +23,8 @@ source "${EXE_ROOT}"/utils.sh
 
 function usage() {
   echo "This script will deploy KubeFin components on primary cluster"
-  echo "Usage: hack/install-kubefin-primary.sh <CLUSTER_PROVIDER> <CLUSTER_NAME> <MULTI_CLUSTER_ENABLE> <KUBEFIN_VERSION>"
-  echo "Example: hack/install-kubefin-primary.sh default kubefin-server true latest"
+  echo "Usage: hack/init_primary_config.sh <CLUSTER_PROVIDER> <CLUSTER_NAME> <MULTI_CLUSTER_ENABLE> <KUBEFIN_VERSION>"
```
Comment (mrhello369) [inline hack/init-primary-config.sh:26]: it should be init-primary-config.sh

Comment (mrhello369) [reply hack/init-primary-config.sh:26]: Same as the hack/cherryxxx，can u make the naming format unified

Comment (oliviershrek) [reply hack/init-primary-config.sh]: cc @mrhello369

**Review by mrhello369 — APPROVED** [review]

**Pull request merged**